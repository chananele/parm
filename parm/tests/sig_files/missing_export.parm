---
name: missing_export
exports:
  - exported
  - unexported
method: find_single
pattern: |
  exported: bne #0x1043c
...
