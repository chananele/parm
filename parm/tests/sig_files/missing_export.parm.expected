---
name: missing_export
result: pass
errors:
  - symbol [unexported] was not exported
matches:
  exported: 0x10458
...


